Sources/NIOCore/AsyncSequences/NIOAsyncWriter.swift

```diff
@@ -111,7 +109,7 @@ public struct NIOAsyncWriterError: Error, Hashable, CustomStringConvertible {
 
     @inlinable
     public var description: String {
-        "NIOAsyncWriterError.\(self._code.rawValue)"
+      "\(self.file): \(self.line) NIOAsyncWriterError.\(self._code.rawValue)"
```

Comment:
We normally stick file and line at the end: as "<file>:<line>". Also, four spaces of indentation please

Reply:
Just to be sure, do you mean the format should be like this "NIOAsyncWriterError.\(self._code.rawValue) \(self.file):\(self.line)"?